### PR TITLE
EJDOTNETCORE-4308 Improvise OCR UG documentation

### PR DESCRIPTION
**Pull request by jeyalakshmit**
Hi sir,

I have improvise OCR UG documentation for the below platforms. Kindly review this and merge. 

Blazor 
ASP.NET Core 
ASP.NET MVC
Azure

Regards,
Jeyalakshmi T

### CODE REVIEW

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Perform OCR on a PDF document in ASP.NET MVC | Syncfusion
+description: Learn how to efficiently perform OCR on a PDF document in ASP.NET MVC with easy steps using Syncfusion .NET library.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/aspnet-mvc.md:3]: Learn how to perform OCR on scanned PDF documents and images with different tesseract version in ASP.NET MVC using Syncfusion .NET OCR library.

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+platform: file-formats
+control: PDF
+documentation: UG
+keywords: Assemblies
+---
+
+# Perform OCR on a PDF document in ASP.NET MVC
+
+The Syncfusion Essential PDF is a .NET PDF library that supports OCR by using the Tesseract open-source engine in ASP.NET MVC application.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/aspnet-mvc.md:12]: The Syncfusion .NET OCR library used to extract text from scanned PDFs and images in ASP.NET MVC application with the help of Google's [Tesseract](https://github.com/tesseract-ocr/tesseract) Optical Character Recognition engine.

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+control: PDF
+documentation: UG
+keywords: Assemblies
+---
+
+# Perform OCR on a PDF document in ASP.NET MVC
+
+The Syncfusion Essential PDF is a .NET PDF library that supports OCR by using the Tesseract open-source engine in ASP.NET MVC application.  
+
+## Steps to perform OCR on a PDF document in ASP.NET MVC
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/aspnet-mvc.md:14]: Steps to perform OCR on entire PDF document in ASP.NET MVC

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Step 1: Create a new C# ASP.NET Web Application (.NET Framework) project.
+   <img src="OCR-Images/aspnetmvc1.png" alt="convert_OCR_ASP.NET_MVC1" width="100%" Height="Auto"/>
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/aspnet-mvc.md:17]: Remove the space before the image tag

File: File-Formats/PDF/Working-with-OCR/blazor.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+         "saveAsFile",
+         filename,
+         Convert.ToBase64String(data));
+}
+
+{% endhighlight %}
+
+Step 12: Add the following JavaScript function in the _Host.cshtml available under the Pages folder.
+
+{% highlight c# tabtitle="C#" %}
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/blazor.md:150]: check the code indentation properly

File: File-Formats/PDF/Working-with-OCR/net-core.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
+---
+title: Perform OCR on a PDF document in ASP.NET Core | Syncfusion
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-OCR/net-core.md:2]: Change the content as like the comments in ASP.NET MVC

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
+---
+title: Perform OCR on a PDF document in ASP.NET MVC | Syncfusion
```
[the remark this answers is not in the document]
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/aspnet-mvc.md]: changed title

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md (hunk 1 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/aspnet-mvc.md]: changed description

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+description: Learn how to efficiently perform OCR on a PDF document in ASP.NET MVC with easy steps using Syncfusion .NET library.
+platform: file-formats
+control: PDF
+documentation: UG
+keywords: Assemblies
+---
+
+# Perform OCR on a PDF document in ASP.NET MVC
```
[the remark this answers is not in the document]
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/aspnet-mvc.md]: title changed

File: File-Formats/PDF/Working-with-OCR/aspnet-mvc.md (hunk 3 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/aspnet-mvc.md]: changed the title

File: File-Formats/PDF/Working-with-OCR/net-core.md (hunk 6 above)
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/net-core.md]: Changed the content

File: File-Formats/PDF/Working-with-OCR/net-core.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+The Syncfusion Essential PDF is a .NET PDF library that supports OCR by using the Tesseract open-source engine in ASP.NET Core application. 
+
+## Steps to perform OCR on a PDF document in ASP.NET Core application
+
+Step 1: Create a new C# ASP.NET Core Web Application project.
+   <img src="OCR-Images/aspnetcore1.png" alt="convert_OCR_ASPNET_CORE1" width="100%" Height="Auto"/>
+
+Step 2:  In configuration windows, name your project and select Next.
+   <img src="OCR-Images/aspnetcore2.png" alt="convert_OCR_ASPNET_CORE2" width="100%" Height="Auto"/>
```
[the remark this answers is not in the document]
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/net-core.md]: removed the space

File: File-Formats/PDF/Working-with-OCR/blazor.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
+---
+title: Perform OCR on a PDF document in Blazor | Syncfusion
```
[the remark this answers is not in the document]
Comment (jeyalakshmit) [reply File-Formats/PDF/Working-with-OCR/blazor.md]: modified the content

**Pull request closed without merging**